openupgrade_scripts/scripts/payment/17.0.2.0/noupdate_changes.xml

```diff
@@ -1,5 +1,355 @@
 <?xml version='1.0' encoding='utf-8'?>
 <odoo>
+    <record id="payment_method_bancontact" model="payment.method">
```

Comment:
add a note saying that these changes are generated manually.

Reply:
done, and I moved the manual changes to a `_work` file as proposed in https://github.com/OCA/OpenUpgrade/issues/4707#issuecomment-2610570291